### DISCUSSION TITLE
Missing meta data

### DISCUSSION THREAD

**Discussion by pinionless** [immich-app/immich#5749] [Q&A] [answered]
For some strange, unknown reason by Pixel 7 Pro does not always save metadata. Its mostly in videos. So I am missing location and timezone.
Any suggestions what would be the easiest way to find those files?
Those files end up with a wrong file name - by 1 hour. 

In Immich I see for example:
Tue, 9:53 PM GMT+01:00 - correct file name 12-12-2023 21.53.27.jpg
Tue, 8:49 PM UTC - wrong file name  12-12-2023 20.49.06.mp4

**Comment by zackpollard** [comment]
Are these videos taken using the normal pixel camera and are they being uploaded from the Immich android app?

**Reply by pinionless** [reply]
I am using stock camera app and Immich app auto backup.
Random videos have no metadata.
Now I see that video from example above has location in Google Photos app but not in Immich after upload.

Just deleted the video from Immich server and backed it up again with the app. Still no location and timezone info extracted from the file.
I could check logs. Server container does exif extraction? What log level to use?

Found another file with correct location information in Google Photos and no exif data in Immich.

I am thinking about editing the immich database to change UTC to GMT+1. easy to find assets with this issue there.
Any way I can easily check if the exif info exists in that video on windows?

**Comment by pinionless** [comment]
I just checked with ffprobe the file that has no exif information in Immich.
And it looks like its all there and for some reason was not extracted.

**Comment by pinionless** [comment, marked as answer]
Found the problem:
All files without exif are bigger than 2 GB
And the logs show at the end of exif extraction:
> "Warning": "End of processing at large atom (LargeFileSupport not enabled)"

Same issue here: https://github.com/immich-app/immich/issues/4349
